### DISCUSSION TITLE
spacy-lookup vs entity ruler ?

### DISCUSSION THREAD

**Discussion by zhangweida2080** [explosion/spaCy#10984] [Help: Coding & Implementations] [answered]
spacy-lookup and entity ruler can do dictionary-based NER, but which should I use? 
Thank you so much.

**Comment by polm** [comment, marked as answer]
[spacy-lookup](https://spacy.io/universe/project/spacy-lookup) is a third-party module that's only for v2 of spaCy that hasn't been updated since 2019. I am not sure but I think it might have been faster than the EntityRuler when released, or maybe even be older than the EntityRuler entirely. 

For new projects though, I think you should just use the EntityRuler.
